scripts/irods/test/test_igroupadmin.py

```diff
@@ -72,6 +71,63 @@ def tearDownClass(self):
             self.admin.__exit__()
             admin_session.assert_icommand(['iadmin', 'rmuser', self.admin.username])
 
+    def test_create_user_with_password_in_igroupadmin__6887(self):
+        USERS = { 'alice':42,
+                  'bob':40, 'eve': 3 }
+
+        # TODO -- eventually delete , but using for test r.n.
+        #obj_name = '{}/{}'.format(self.groupadmin.home_collection, 'test__6887__object')
+        #self.groupadmin.assert_icommand(['ichmod', 'write', self.groupadmin.username, obj_name], 'EMPTY' )
+
+        for username,pwlen in USERS.items():
+            try:
+                passwd = (username[0]+'pass_6887_the_quick_brown_fox_jumps_over_the_lazy_dog')[:pwlen]
+                self.groupadmin.assert_icommand(['igroupadmin', 'mkuser', username, passwd], 'EMPTY' )
+                with session.make_session_for_existing_user(username, passwd, lib.get_hostname(), self.admin.zone_name):
+                    pass
+            finally:
+                self.admin.run_icommand(['iadmin','rmuser',username])
+
+    def test_remove_grom_group_scenarios__6888(self):
+        """Thoroughly test atg, rfg (remove from group) and transfer of responsiblity to another group administrator"""
+        self.rodsuser_name = 'alice'
+        try:
+            # Use the groupadmin account to create group and set itself up as administrator.
+            self.groupadmin.assert_icommand( ['igroupadmin', 'mkgroup', self.group ], 'EMPTY' )
```

Comment:
Does the group variable need to be attached to the class instance / self rather than being a normal local variable?

Also, I don't see where it is defined.

Reply:
for efficiency, I suppose.  It's assigned in `setupClass`, which I had nothing to do with....

Reply:
All good. I somehow missed the initialization of `self.group`.

Please mark as resolved.